### PR TITLE
yusuke.shirakawa のBANを解除する

### PR DESCRIPTION
**Pull request by fumihiroito**
CC: @ozyozyo

### CODE REVIEW

File: imghr.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func main() {
 		}
 	}
 	log.Print("init")
-	ignoreUsers := []string{"U037G7YJF"}
+	ignoreUsers := []string{}
```
Comment (ozyozyo) [inline imghr.go]: @shirochan 
Are you sure it's all right :confused: ?

**Comment by shirochan** [comment]
Trust me!

**Comment by ozyozyo** [comment]
LGTM :ok_woman:

**Comment by f110** [comment]
ok, accept.

**Pull request merged**